test/k8sT/KafkaPolicies.go

```diff
@@ -92,6 +93,9 @@ var _ = Describe("K8sKafkaPolicyTest", func() {
 				if !res.WasSuccessful() {
 					return false
 				}
+				if strings.Contains(res.GetStdOut(), "can't resolve") {
```

Comment:
The example output in the PR description shows that the "can't resolve" is in the `stderr`, not the `stdout` of the command; should we check both `stdout` and `stderr`?